### ISSUE TITLE
VoiceOver doesn’t read username for each posts.

### ISSUE THREAD

**Issue by veer66** [DagAgren/toot#248]
VoiceOver doesn’t read username for each posts. So I can hear only content but I don’t know who posted it.

**Comment by chucker** [comment]
See #123
